tee-worker/local-setup/github-action-config-rococo.json

```diff
@@ -15,7 +15,9 @@
         "-u",
         "wss://parachain-eu-0.staging.litentry.io",
         "-p",
-        "443"
+        "443",
+        "running-mode",
+        "stage"
```

Comment:
I think `-u` and `-m` can be confusing/conflicting: can I use `-u wss://parachain-eu-0.staging.litentry.io` and `-m dev` at the same time? Again it depends what are `dev/staging/prod` supposed to do

Reply:
I think there is no shortcut for 'running-mode'. Always need to type the full name. I also feel confusing about these parameter shortcuts. That's the reason NO shortcut provided here.